python/tk_hiero_export/version_creator.py

```diff
@@ -356,6 +356,13 @@ def finishTask(self):
             version_data=vers,
         )
 
+        # Log usage metrics
+        try:
+            self.app.log_metric("Version Create", log_version=True)
+        except:
+            # ingore any errors. ex: metrics logging not supported
+            pass
+
 
```

Comment:
Do we want to also track whether the transcode just published images or if it also created a Version in Shotgun? Might be overkill. But the "Version Create" name is misleading as they may not have actually created a Version here. They may have just published their transcoded images.